Sources/FluentKit/Query/Builder/QueryBuilder+Paginate.swift

```diff
@@ -80,6 +86,20 @@ public struct PageMetadata: Codable {
         self.page = page
         self.per = per
         self.total = total
+        
+        // Find out how many items are left in the pagination object
+        let remainingItems = total - (per * page)
+        if remainingItems <= 0 {
+            self.hasNext = false
+        } else {
+            self.hasNext = true
+        }
```

Comment:
How do we feel about one-liners like below?

```suggestion
        self.hasNext = (total - (per * page)) <= 0
        self.hasPrevious = page > 1
```

Reply:
Personally I'd choose the existing code, it's a bit easier to read at a glance. But I don't have a strong opinion on it